### DISCUSSION TITLE
how to set proxy value in route

### DISCUSSION THREAD

**Discussion by zhangshunbao** [apache/apisix#5570] [Q&A]
In nginx, we can set proxy value in context  http/server/location, like this:
            proxy_cache off;
            proxy_buffer_size 64k;
            proxy_buffers   32 32k;
            proxy_busy_buffers_size 128k;
            fastcgi_buffer_size 256k;
            fastcgi_buffers 32 256k;
            proxy_connect_timeout 600;
            proxy_send_timeout 600;
            proxy_read_timeout 600;
how to set those value in apisix route?

**Comment by tokers** [comment]
Some of them cannot be set dynamically, If I am correct, the following items can be set on the route/upstream level:

1. proxy_cache, through the [proxy-cache plugin](http://apisix.apache.org/docs/apisix/plugins/proxy-cache);
2. proxy_*_timeout, through Route or Upstream, see http://apisix.apache.org/docs/apisix/admin-api.
